Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -198,16 +200,52 @@ private void Menu_Loaded(object sender, RoutedEventArgs e)
                 _wrapPanel.Orientation = Orientation;
             }
 
+            Window.Current.CoreWindow.PointerMoved -= CoreWindow_PointerMoved;
             LostFocus -= Menu_LostFocus;
             LostFocus += Menu_LostFocus;
             Dispatcher.AcceleratorKeyActivated -= Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown -= CoreWindow_KeyDown;
             Dispatcher.AcceleratorKeyActivated += Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown += CoreWindow_KeyDown;
+            Window.Current.CoreWindow.PointerMoved += CoreWindow_PointerMoved;
+        }
+
+        private void Menu_LayoutUpdated(object sender, object e)
```

Comment:
LayoutUpdated is a deceiving event. You would think it would only fire when the control in question changes. Best to put a Debug.Writeline in the event and see exactly when it changes when doing various things within an app. I've been away from a computer for four days but I can post a sample at some point this week